src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
@cindyli it is not currently used, but is there to provide a similar function as `fl-input-enhance`, that is to provide a way for an integrator to enhance custom made buttons.